Stripe/STPCard.m

```diff
@@ -84,19 +87,34 @@ + (NSString *)stringFromBrand:(STPCardBrand)brand {
     }
 }
 
+#pragma mark - STPCardFundingType
+
++ (NSDictionary <NSString *, NSNumber *> *)stringToFundingMapping {
+    return @{
+             @"credit": @(STPCardFundingTypeCredit),
+             @"debit": @(STPCardFundingTypeDebit),
+             @"prepaid": @(STPCardFundingTypePrepaid),
+             @"other": @(STPCardFundingTypeOther),
```

Comment:
I might need to do some cleanup here to align with what our API does:

<img width="580" alt="screen shot 2017-06-21 at 3 03 11 pm" src="https://user-images.githubusercontent.com/28276156/27408720-e4dafd8e-5692-11e7-91d9-2b06ce59ea0d.png">

https://stripe.com/docs/api#card_object-funding